docs/frontends/tvm-relay.md

```diff
@@ -1,11 +1,13 @@
-TVM to Calyx via Relay
-======================
+# TVM Relay
 
-This is an in-progress compiler from [TVM][]'s intermediate representation, [Relay][], targeting Calyx.
+[TVM][] is a compiler for machine learning frameworks that can 
+optimize and target kernels to several different backends.
 
+The TVM Relay frontend lives in the [relay-lang][] folder in the
+Calyx repository and generates Calyx components from the Relay
+intermediate representation.
```

Comment:
Super minor, but I did like the above deleted explanation of what Relay is:

> [TVM][]'s intermediate representation, [Relay][], […]

Reply:
Updated - I'll let you OK it.

Reply:
LGTM! Merge it